en/developer_guide/fieldType.rst

```diff
@@ -0,0 +1,101 @@
+Field type
+==========
+
+Open Orchestra offers a RAD approach to create the Content Types.
+Further information about it are available in the `Content Types documentation`_.
+
+During creating a content type with different fields, you can choose between different types.
+By default Open Orchestra offers few types (text line, date, email, hidden, integer, choice, media, money, rich test, text area).
+
+
+.. image:: ../../images/fieldType.png
+
+Add a field type
+----------------
+To add a new field type you can add in your configuration parameters an entry
+to the ``open_orchestra_backoffice.field_types`` key
+which describes your field type.
+
+The description of your field type must contains these values:
+
+* ``name`` name of your field type
+* ``type`` type of your field, this can be a type natively available in Symfony or a custom type
+* ``label`` translation key of label
+* ``default_value`` (optional) describes the default value field
+* ``options`` (optional) describes options of your field type
+
+``default_value`` and ``options`` are describe in more detail in subsequent sections.
+
+Example of basic field type:
+
+.. code-block:: yaml
+
+    open_orchestra_backoffice.field_types:
+        custom_textarea:
+            label: open_orchestra.form.field_type.textarea
+            type: textarea
+
+Field type options
+------------------
+
+The field type can be configure with options, for instance to add an option
+to field ``custom_textarea`` to configure the max length.
+
+You must add an entry in ``open_orchestra_backoffice.options`` key with 4 parameters:
+
+* ``name`` name of option
+* ``type`` type of option, this can be a type natively available in Symfony or your custom type
+* ``label`` translation key of label
+* ``required`` a boolean who indicate if option is required
+
+Example of option ``max length``:
+
+.. code-block:: yaml
+
+    open_orchestra_backoffice.options:
+        max_length:
+            type: integer
+            label: open_orchestra_backoffice.form.orchestra_fields.max_length
+            required: true
+
+Now, we can add the option in your field type:
+
+.. code-block:: yaml
+
+    custom_textarea:
+        label: open_orchestra.form.field_type.textarea
+        type: textarea
+        options:
+            max_length:
+                default_value: 100
+
+The parameter ``default_value`` is require in all options .
+
+Open Orchestra offers already many options (max_length, required, grouping, rounding_mode, multiple, expanded
+choices, currency, precision, format, widget, input).
+
+Field type default value
+------------------------
+
+Open Orchestra offers the possibility to add a field for add a default value when your field type is display.
```

Comment:
I'll say that on another way: when adding an attribute to a content type, the user may contribute the default value for this attribute. As the default value type depends on the field type (datetime vs textarea), it's interesting to personnalize the default value form.